### ISSUE TITLE
issue with uk site

### ISSUE THREAD

**Issue by ColinMoulds** [nodejs/nodejs.org#2069]
i wasn't sure what nodejs/ to use but https://nodejs.org/uk/ is in russian for no apparent reason!

**Comment by fhemberger** [comment]
@ColinMoulds 'uk' = Ukranian 😂

**Comment by ColinMoulds** [comment]
@fhemberger its the site i get redirected to both here and in sweden lol

**Comment by fhemberger** [comment]
Strange, what browser are your using and what are your language settings?
British English should be `en_GB`, so I don't know why it would redirect to uk.

(In the meantime, https://nodejs.org/en/ should help 😉)

/cc @nodejs/build

**Comment by ghost** [comment]
@ColinMoulds：You can use Google chrome to have a play, and everything seems fine with me :)

**Comment by refack** [comment]
IIUC this is the magic souse:
```
    location / {
        rewrite ^/$ /en/ redirect;

        try_files $uri $uri/ @english_fallback;

        location ~ \.json$ {
            add_header access-control-allow-origin *;
        }
    }

    # instead of serving a 404 page when a page hasn't been translated
    location @english_fallback {
        if ($uri ~* ^/(ar|ca|de|es|fa|fr|gl|it|ja|ko|pt-br|uk|zh-cn|zh-tw)/) {
            set $lang $1;
        }
        rewrite ^/(ar|ca|de|es|fa|fr|gl|it|ja|ko|pt-br|uk|zh-cn|zh-tw)/(.*)$ /en/$2;
    }
```
tl;dr default to `/en/` and fall back to `/en/` in case translated file does not exist 🤷‍♂️

**Comment by ColinMoulds** [comment]
I was using Mozilla Firefox Nightly

**Comment by ghost** [comment]
@ColinMoulds：What about using the latest stable version of FF？Is it still happening?

**Comment by alexandrtovmach** [comment]
Should be closed
